src/main/java/frc/robot/subsystems/shooter/ShooterBed.java

```diff
@@ -0,0 +1,68 @@
+package frc.robot.subsystems.shooter;
+
+import org.littletonrobotics.junction.Logger;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class ShooterBed extends SubsystemBase
+{
+    private ShooterBedIO                       _io;
```

Comment:
Let's make this final:

```suggestion
    private final ShooterBedIO                       _io;
```